sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -314,6 +314,27 @@ case class Divide(left: Expression, right: Expression) extends DivModLike {
   override def evalOperation(left: Any, right: Any): Any = div(left, right)
 }
 
+@ExpressionDescription(
+  usage = "a _FUNC_ b - Divides a by b.",
```

Comment:
nit: explicitly say this is integral divide?

Reply:
yes, thanks, I am very bad at descriptions.